appmap/_implementation/recording.py

```diff
@@ -355,7 +355,18 @@ def wrapped_find_spec(find_spec, _, args, kwargs):
     if spec is not None:
         if getattr(spec.loader, "exec_module", None) is not None:
             loader = spec.loader
-            loader.exec_module = wrap_exec_module(loader.exec_module)
+            # This is kind of gross. As this comment linked to below describes, wrapt has trouble
+            # identifying methods decorated with @staticmethod. It offers two suggested fixes:
+            # update the class definition, or patch the function in __dict__. We can't do the
+            # former, so do the latter instead.
+            #   comment in wrapt: https://github.com/GrahamDumpleton/wrapt/blob/master/src/wrapt/wrappers.py#L730
+            #
+            # TODO: determine if we can use wrapt.wrap_function_wrapper to simplify this code
+            exec_module = inspect.getattr_static(loader, "exec_module")
+            if isinstance(exec_module, staticmethod):
+                loader.exec_module = wrap_exec_module(exec_module)
```

Comment:
I don't entirely understand how it's different. Surely if `exec_module` is a `staticmethod`, `loader.exec_module === exec_module`? Or is there something else going on?

Reply:
Ah, never mind, actually `loader.exec_module == exec_module.__wrapped__` (or whatever is the attr called).

Reply:
Yeah:
```
ajp@Alans-MacBook-Pro appmap-python % python
Python 3.10.7 (main, Sep 15 2022, 04:53:36) [Clang 13.1.6 (clang-1316.0.21.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib.machinery import BuiltinImporter
>>> import inspect
>>> getattr(BuiltinImporter, 'exec_module')
<function BuiltinImporter.exec_module at 0x10edde4d0>
>>> inspect.getattr_static(BuiltinImporter, 'exec_module')
<staticmethod(<function BuiltinImporter.exec_module at 0x10edde4d0>)>
```